docs/guides/modules/test/pages/fix-flaky-tests.adoc

```diff
@@ -425,7 +425,7 @@ If you get the following error inside a Chunk task, this indicates that your Ope
 Action required - agent execution error
 The agent ran into an error while executing this task. See our community forum for how to solve this error.
 
-Contact mailto:sebastian@circleci.com[sebastian@circleci.com] for assistance.
+Have feedback or feature requests? Submit them on our [Canny board](https://circleci.canny.io/cloud-feature-requests?selectedCategory=chunk-ai) where you can also see what others in the community are requesting.
```

Comment:
Seems like this should be somewhere more obvious on the page not in this troubleshooting section. Mind if I move it @luisejroblesci ?